app/scripts/model/dataProxy.js

```diff
@@ -371,15 +251,21 @@ window.DataManagerForIviz = (function($, _) {
                 _metaObj.filter = [];
                 _metaObj.keys = {};
                 _metaObj.numOfDatum = 0;
-                _metaObj.priority = -1;
-                _metaObj.show = true;
+                if (!_.isArray(_metaObj.priority)) {
+                  iViz.priorityManager
+                    .setClinicalAttrPriority(_metaObj.attr_id, [1, 1, 1, Number(_metaObj.priority)]);
```

Comment:
But you still need to loop through all clinical attributes to get these priorities?